packages/larva-svg/README.md

```diff
@@ -1,27 +1,169 @@
+1. Indivdual, optimized SVG icons
+1. A pre-built SVG sprite to use along with the `c-icon` Larva pattern
+1. A script to build SVG sprites at the project-level for brand-specific SVG assets
 
-**Icons are available with both fragment identifiers and as inline SVGs. Logos are only available as inline SVG.**
+## Links
 
-## Notes
+* [Icons available in this package](https://penske-media-corp.github.io/pmc-larva/packages/larva-svg/build/defs/sprite.defs.html)
+* [svg-sprite npm package documentation](https://github.com/jkphl/svg-sprite)
+* [CSS-Tricks article about SVG sprites](https://css-tricks.com/svg-sprites-use-better-icon-fonts/)
 
-Icons are available as SVG fragments to be accessed with `a-icon` in CSS and as inline SVG, accessed with `c-icon` and displayed in the `use` element.
+## Overview of Functionality
 
-In order for inline SVG with `c-icon` to work, the defs.svg must be inlined and hidden on the page. For the stack, or CSS icons with `a-icon` to work, the stack.svg must be available for retrieval from the stylesheet.
+**This package uses the low-level svg-sprite npm package to build an SVG sprite from a directory of SVG files**. It can be used to build sprites in this repository, or in a consuming project. 
 
-More information here: [SVG Fragment Identifiers](https://css-tricks.com/svg-fragment-identifiers-work/).
+1. Given a directory of SVG files stored inside `./src/svg/`
+2. `index.js` is exectuted as a node script from `./`
+3. All files in `./src/svg/` are piped into the svg-sprite npm package
+4. The SVGs are optimized via SVGO
+5. The SVGs are compiled into a single sprite file
+6. The package's output is written to `./build/defs/`
 
-## Adding new icons and logos
+**Important note:**
+
+The consuming project is responsible for: 
+
+1. Loading the sprite on the front-end.
+2. Providing a script to combine the from larva-svg with a local SVG sprite, if applicable.
+
+Examples are provided in this documentation, but this package does not provide the above functionality.
+
+## Development Setup
+
+### To copy the icon sprite from this package into a consuming project:
+
+1. Install the package. Run this command from the same location as package.json.
+	```language:bash
+	npm install @penskemediacorp/larva-svg --save
+	```
+
+2. Create a directory `./build/svg`. (where `./` is the root of `assets` or wherever your project stores front-end files)
+
+3. Add the following copy script to the local project's package.json:
+
+	```language:json
+	"scripts": {
+		"update-icons": "cp ./node_modules/@penskemediacorp/larva-svg/build/defs/svg/sprite.defs.svg ./build/svg",
+	}
+	```
+
+4. Run `npm run update-icons` to copy the sprite to your local project.
+
+5. Consider using the following tools to load the sprite on the front-end: 
```

Comment:
For someone like me who is very new to this, I think it might be helpful if you have a link to an example of how this is used. Or just a bit more information on once everything is setup, how you then use them within your project. I wouldn't know where to go once I had it all setup. Just a thought, let me know what you think!

Reply:
Yes, excellent thought! This is a kind of obscure bit of knowledge to assume - will add a basic "What / Why SVG Sprites" at the beginning. Thanks!!